{"level":50,"time":"24/12/2021, 18:56:32","pid":2616,"hostname":"DESKTOP-5T9QAVF","err":{"type":"Error","message":"Ops","stack":"Error: Ops\n    at Object.getBoards (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\src\\resources\\boards\\boards.controllers.ts:13:9)\n    at preHandlerCallback (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\handleRequest.js:126:28)\n    at next (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:158:7)\n    at Object.<anonymous> (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\src\\logger\\logger.ts:23:5)\n    at hookIterator (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:237:10)\n    at next (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:164:16)\n    at hookRunner (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:187:3)\n    at preValidationCallback (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\handleRequest.js:101:5)\n    at handler (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\handleRequest.js:72:7)\n    at handleRequest (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\handleRequest.js:20:5)"},"msg":"Ops"}
{"level":50,"time":"24/12/2021, 19:16:36","pid":17132,"hostname":"DESKTOP-5T9QAVF","err":{"type":"DatabaseError","message":"No response received from the database, please try later","stack":"Error: No response received from the database, please try later\n    at RepositoryMaintenance.getBd (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\src\\bd\\repository_maintenance.ts:35:13)\n    at Object.getBoards (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\src\\resources\\boards\\boards.controllers.ts:13:21)\n    at preHandlerCallback (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\handleRequest.js:126:28)\n    at next (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:158:7)\n    at Object.<anonymous> (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\src\\logger\\logger.ts:23:5)\n    at hookIterator (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:237:10)\n    at next (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:164:16)\n    at hookRunner (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\hooks.js:187:3)\n    at preValidationCallback (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\handleRequest.js:101:5)\n    at handler (D:\\Work\\Task-4.-REST-service\\nodejs2021Q4-service\\node_modules\\fastify\\lib\\handleRequest.js:72:7)","statusCode":503},"msg":"No response received from the database, please try later"}
